GSA-TTS/federal-website-standards
# Add the hero component to the homepage

## Context
Fixes #186 

## Description
This adds the hero component to the base layout if the current page permalink is `/` so that the hero only renders on the homepage. There's an included overrides file to make the style fit the mockup.

## How to verify this change
[Visit the preview URL](https://federalist-142078a8-f654-4daa-8f73-db9770b3ec7a.sites.pages.cloud.gov/preview/gsa-tts/federal-website-standards/caley/hero/) and observe the hero component below the nav menu on the homepage.
